providers/base/units/mtd/manifest.pxu

```diff
@@ -1,4 +1,4 @@
 unit: manifest entry
 id: has_mtd
-_name: Does platfrom support Memory Technology Device?
+_name: Memory Technology Device (MTD) support?
```

Comment:
I think the last part (`support?`) is not required here, because there is already a question above in the Manifest screen:

> Does this machine have this piece of hardware?
> 
> Memory Technology Device (MTD) support?

It feels better if it's just

> Does this machine have this piece of hardware?
> 
> Memory Technology Device (MTD)